src/edfi-paging-test/edfi_paging_test/reporter/measurement.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+from typing import Callable, Tuple, TypeVar
+import timeit
+
+T = TypeVar("T")
+
+
+@dataclass
+class Measurement:
+    resource: str
+    URL: str
+    page_number: int
+    page_size: int
+    number_of_records: int
+    elapsed_time: float
+    http_status_code: int
+
+    @staticmethod
+    def timeit(callback: Callable[[], T]) -> Tuple[float, T]:
```

Comment:
timeit static method feels a bit misplaced inside Measurement class. Consider moving it to out to keep Measurement a simple data class.